models/auth/token_scope.go

```diff
@@ -250,7 +250,7 @@ func (s AccessTokenScope) parse() (accessTokenScopeBitmap, error) {
 			remainingScopes = remainingScopes[i+1:]
 		}
 		singleScope := AccessTokenScope(v)
-		if singleScope == "" {
+		if singleScope == "" || singleScope == "sudo" {
```

Comment:
```suggestion
		if singleScope == "" || singleScope == "sudo" { // sudo was recognized previously but isn't anymore
```

Reply:
By this logic, wouldn't we be ignoring all previous scopes that we no longer support?